src/main/java/org/faceletslite/imp/FaceletsCompilerImp.java

```diff
@@ -32,6 +32,7 @@
 import javax.xml.transform.dom.DOMSource;
 import javax.xml.transform.stream.StreamResult;
 
```

Comment:
Das `reset()` in Zeile 114 ist ein bisschen merkwürdig. Damit wird die Konfiguration aus `DefaultConfiguration.createDocumentTransformer()` ausgehebelt.

Reply:
Oha. Natürlich muss die default configuration erhalten bleiben. Falls das nicht so ist, sind die beiden resets falsch. 
Das würde einiges erklären...